src/main/java/io/jenkins/plugins/synopsys/security/scan/service/bridge/BridgeDownloadParametersService.java

```diff
@@ -143,7 +143,12 @@ public String getPlatform() {
         if (os.contains("win")) {
             return ApplicationConstants.PLATFORM_WINDOWS;
         } else if (os.contains("mac")) {
-            return ApplicationConstants.PLATFORM_MAC;
+            String arch = Utility.getAgentOsArch(workspace, listener);
+            if (arch.startsWith("arm") || arch.startsWith("aarch")) {
+                return ApplicationConstants.PLATFORM_MAC_ARM;
+            } else {
+                return ApplicationConstants.PLATFORM_MACOSX;
+            }
```

Comment:
@zaman-akib Please Cover these lines(146-150) in unit tests.

Reply:
Covered